portal/src/main/webapp/js/api/cbioportal-datamanager.js

```diff
@@ -953,6 +953,31 @@ window.initDatamanager = function (genetic_profile_ids, oql_query, cancer_study_
 	'getGeneticProfileIds': function () {
 	    return this.genetic_profile_ids;
 	},
+	/**
+	 * Selects the genetic profile most suitable for heatmap rendering.
+	 *
+	 * @returns {Promise<?string>} promise resolving with the
+	 * stable ID of the profile, or with null if no applicable
+	 * profiles are in use.
+	 */
+	'getDefaultHeatmapProfile': function () {
```

Comment:
I addressed this in https://github.com/cBioPortal/cbioportal/pull/1847/commits/86ece36645fdaa1f43266ea7c5e228788ac344a7